syft/pkg/cataloger/alpm/parse_alpm_db.go

```diff
@@ -31,8 +31,13 @@ var (
 	}
 )
 
+type parsedData struct {
+	Licenses         string `mapstructure:"licenses"`
```

Comment:
```suggestion
	Licenses         string `mapstructure:"license"`
```
This is currently missing existing licenses that the tests already captured (the tests data for this seem to be removed)